README.md

```diff
@@ -29,56 +29,25 @@ Description of the Erica connector:
 More information can be found here (German):  
 [http://hc-ddr.hucki.net/wiki/doku.php/z9001/erweiterungen/s3004](http://hc-ddr.hucki.net/wiki/doku.php/z9001/erweiterungen/s3004)
 
-A schematic of our Arduino based interface can be found [on EasyEDA](https://easyeda.com/sirexeclp/erikaarduinointerface).
-![Schematic](docs/Schematic_ErikaArduinoInterface.png)
+A schematic of our Raspberry Pi based interface can be found [on EasyEDA](https://easyeda.com/sirexeclp/erikaraspberrypiinterface).
+![Schematic](docs/Schematic_ErikaRaspberryPiInterface.png)
 
 [DDR-Halbleiter - Kurzdatenblätter und Vergleichsliste](https://www-user.tu-chemnitz.de/~heha/basteln/Konsumg%C3%BCter/DDR-Halbleiter/)
-## Software
 
-### Arduino Sketch
-
-Located in `arduino` directory.
-
-The arduino sketch should be compilable for any Arduino that has at least one hardware 
-[UART](https://en.wikipedia.org/wiki/Universal_asynchronous_receiver-transmitter) (hardware serial).  
-TODO: Test on Mini, Nano
-
-The arduino will communicate with your PC using the default hardware serial, which is usually connected to a 
-USB-to-serial converter.  
-It will use a software serial to communicate with the Erika3004.  
-More information about SoftwareSerial can be found on the 
-[Arduino-Website](https://www.arduino.cc/en/Reference/SoftwareSerial).
-In `erika.ino`, you can set the following constants:
-
-> Remember: Serial connections are connected with cross-over. This means RX on the Arduino connects to TX on the Erika and vice versa.
-
-|Name|Description|Default Value|Comment|
-|----|-----------|-------------|-------|
-|PC_BAUD| Baudrate used to communicate with the pc|9600|You can change this to fit your needs.|
-|RTS_PIN| Pin that is connected to the ready to send pin of the Erika.|3|You can change this to any available digital input pin on your arduino.|
-|ERIKA_RX| Recieve pin of the SoftwareSerial to communicate with Erika.|10|Check the limitations section on the  [Arduino-Website](https://www.arduino.cc/en/Reference/SoftwareSerial), to find out which pins can be used by SoftwareSerial.|
-|ERIKA_TX| Transmit pin of the SoftwareSerial to communicate with Erika.|11|Check the limitations section on the  [Arduino-Website](https://www.arduino.cc/en/Reference/SoftwareSerial), to find out which pins can be used by SoftwareSerial.|
-|ERIKA_BAUD| Baudrate used by the Erika3004.|1200|DON'T CHANGE THIS! The Erika3004 will only work with 1200 Baud!|
-
-Translation from ASCII to DDR-ASCII is done using arrays defined in `ddr2ascii.h` and `ascii2ddr.h`.  
-
-Let's build a secret chat system using 2 of these. :)
+## Configure Hardware Controlflow
 
+Hardware Controlflow is disribed in the wiki: [Hardware-control-flow-(RTS,-CTS)](https://github.com/Chaostreff-Potsdam/erika3004/wiki/Hardware-control-flow-(RTS,-CTS)).
 
```

Comment:
I would usually caution against a link containing "(" or ")", rather suggesting "%28" and "%29", but it seems to work.